superset-frontend/package.json

```diff
@@ -38,12 +38,14 @@
   ],
   "scripts": {
     "_prettier": "prettier './({src,spec,cypress-base,plugins,packages,.storybook}/**/*{.js,.jsx,.ts,.tsx,.css,.less,.scss,.sass}|package.json)'",
-    "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production BABEL_ENV=\"${BABEL_ENV:=production}\" webpack --mode=production --color",
+    "build": "npm run build-translations && cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production BABEL_ENV=\"${BABEL_ENV:=production}\" webpack --mode=production --color",
```

Comment:
I wouldn't assume that everyone wants translations as it slows down build time and makes the build heavier, especially given the unknown/incomplete state of various translations. I'd add this as an optional step for those interested in it.